Makefile

```diff
@@ -19,7 +19,8 @@ workspace: build
 
 # Experimental Xcode project generator based on Bazel
 workspace_v2:
-	tools/bazelwrapper build :workspace_v2
+	tools/bazelwrapper build -s :workspace_v2 :workspace_v2_xcode_project_deps
+	tools/bazelwrapper build //workspace_v2.xcodeproj/XCHammerAssets:deps
```

Comment:
2nd line is unnecessary, first change is necessary for building the project deps